test/common/swagger_spec.js

```diff
@@ -178,6 +178,7 @@ SwaggerTestSpec.prototype.makeRequest = function(parameters, responseCode) {
 			var req = supertest(__testContext.baseUrl);
 
 			if (self.method === 'post') {
+				headers['Content-Type'] = 'application/json';
```

Comment:
Our API is always dealing with JSON content type. So probably we can set this header for all. 

https://github.com/LiskHQ/lisk/blob/234bf7ec57d8c7a006f7b7d17f40a676d50059ef/test/common/swagger_spec.js#L147

Reply:
It's not a strict requirement for GET requests since they pass data in the `?query` instead of the request body but yes we can do that and should probably do this for `PUT` at least.